app/views/layouts/customer_layout.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CustomerOrder</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <%= yield %>
+  </body>
+  <% if !flash[:alert].nil? %>
+    <p class="my_alert"><%= flash[:alert] %></p>
+  <% end %>
```

Comment:
This block of code outputs the **p tag** outside of the body tag, the code should output **inside the body tag**. Please change this.